tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py

```diff
@@ -581,13 +578,6 @@ def mock_hpu_available(monkeypatch, value=True):
     monkeypatch.setattr(lightning_habana.plugins.precision, "_HPU_AVAILABLE", value)
 
 
-def test_devices_auto_choice_cpu(monkeypatch, cuda_count_0):
```

Comment:
why do we remove this test? is it duplicated somewhere?

Reply:
good call, not sure so rather pull it back :chipmunk: